app/src/main/java/com/github/swent/echo/compose/components/ListDrawer.kt

```diff
@@ -124,7 +127,15 @@ fun EventListItem(
     // Format the association name to be displayed
     val association = event.organizer?.name?.let { "$it • " } ?: ""
     // Format the distance to be displayed
-    val dist = distance?.let { "${it}km • " } ?: ""
+    val dist =
+        distance
+            ?.let {
+                val fmt = { format: String, value: Double ->
+                    String.format(Locale.getDefault(), format, value)
+                }
+                if (it < 1000) fmt("%.0fm", it) else fmt("%.1fkm", it / 1000.0)
```

Comment:
Knitpick: good practise to not Magic values in the code: 1000.0 can be extracted into a variable?